reasoning_gym/algebra/polynomial_equations.py

```diff
@@ -62,6 +62,15 @@ def __init__(self, config: PolynomialEquationsConfig):
             "Determine the real value(s) of {variable} that satisfies: {polynomial_expanded} = 0",
             "Solve the polynomial equation for real {variable}:\n{polynomial_expanded} = 0",
         ]
+        self.added_instruction = """
+        \n\n
+        In solving the equations, please abide by the following instruction:
+        ## 1. All answers should be inserted in square brackets. For example [], [-0.3773, 0.4005] etc.
```

Comment:
The score_answer() method expected simple comma-separated lists, there was a mismatch between the strings in `item["answer"]` and what score_answer() could really accept. I fixed that and added a unit test. No brackets are necessary. If no real solution exists the answer should be an empty string.